Karatsuba_Algo/karatsuba.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python3
+
+from math import ceil, floor
+
+
+def karatsuba(x,y):
+    #base case
+    if x < 10 and y < 10: # in other words, if x and y are single digits
+        return x*y
+
+    n = max(len(str(x)), len(str(y)))
+    m = ceil(n/2)   #Cast n into a float because n might lie outside the representable range of integers.
+
+    a = floor(x / 10**m)
```

Comment:
Operators should have space around them. For more clarity, you can use brackets.